python/packages/autogen-agentchat/tests/test_code_executor_agent.py

```diff
@@ -217,9 +213,13 @@ async def test_self_debugging_loop() -> None:
 
         elif isinstance(message, CodeExecutionEvent) and message_id == 1:
             # Step 2: First code execution
+            execution_text = message.to_text().strip()
+            # Check for key parts of the error message in a platform-agnostic way
             assert (
-                incorrect_code_result in message.to_text().strip()
-            ), f"Expected {incorrect_code_result} in execution result, got: {message.to_text().strip()}"
+                "mean = sum(numbers) / len(numbers" in execution_text
```

Comment:
Hi, @tejas-dharani .
Why did you update this file? it seems this change is not related with your PR.